apps/meteor/ee/app/license/server/getStatistics.ts

```diff
@@ -18,7 +18,7 @@ type ENTERPRISE_STATISTICS = {
 export async function getStatistics(): Promise<ENTERPRISE_STATISTICS> {
 	const statsPms: Array<Promise<any>> = [];
 
-	const statistics: ENTERPRISE_STATISTICS = {} as any;
+	let statistics: ENTERPRISE_STATISTICS = {} as any;
```

Comment:
Every time you type any, a ts dev dies

Reply:
Lol :joy:

Reply:
Fixed those "any" typings on this file :tada: